qutip/core/superop_reps.py

```diff
@@ -232,7 +233,7 @@ def _choi_to_chi(q_oper):
     """
     nq = _nq(q_oper.dims)
     B = _superpauli_basis(nq).data
-    return Qobj(B.adjoint() @ q_oper.data @ B,
+    return Qobj(matmul(matmul(B.adjoint(), q_oper.data), B),
```

Comment:
Pretty sure this operation is called `inner_op` or something on the data layer and is optimised.

Reply:
Doesn't inner_op return a scalar?